tests/medic-conf/medic-conf-actions.spec.js

```diff
@@ -0,0 +1,28 @@
+const {expect} = require('chai');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const url = require('../utils').getInstanceUrl();
+
+//Not all actions tested here due to missing forms and config
+//[convert-collect-forms , upload-collect-form, upload-branding, upload-partners, upload-privacy-policies]
+const actions = ['compile-app-settings','backup-app-settings','convert-app-forms','convert-contact-forms',
+  'backup-all-forms','upload-app-forms','upload-contact-forms', 'upload-resources','upload-custom-translations'];
+
+describe('Medic-conf actions tests', () => {
+  const runCommand = async (action) => {
```

Comment:
In the previous review, I suggested to add a try-catch block here, so we send useful logs to the tests: https://github.com/medic/cht-core/pull/7073#discussion_r621873844

Reply:
done